metadata-jobs/mce-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeProposalsProcessor.java

```diff
@@ -41,6 +43,7 @@
 @RequiredArgsConstructor
 public class MetadataChangeProposalsProcessor {
 
+  @NonNull
```

Comment:
I don't love this annotation because it just throws an NPE still and adds confusion around the javax Nonnull annotation which just adds/removes IDE warnings, but I'm open to being convinced it's useful :P

Reply:
Yeah, I can remove it. I added it because I was getting an error later and was trying to catch it earlier in a more logical spot, earlier in the spring boot initialization. Turns out the authentication filter is required even if authentication is disabled because its used to extract the auditstamp or construct it from that filter. I'll remove this.